packages/unlock-js/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changes
 
+# 0.33.2
```

Comment:
need to add "version": "0.33.2" to `package.json` to match this one

Reply:
@SouravInsights update for you

Reply:
ohh, okay.. i'll update @kalidiagne